cmd/cloud/setup.go

```diff
@@ -308,3 +343,71 @@ func checkAPIKeys(astroClient astro.Client, coreClient astrocore.CoreClient, arg
 	}
 	return true, nil
 }
+
+func checkAPIToken(isDeploymentCmd bool, args []string) (bool, error) {
+	// check os variables
+	astroAPIToken := os.Getenv("ASTRO_API_TOKEN")
+	if astroAPIToken == "" {
+		return false, nil
+	}
+	if !isDeploymentCmd {
```

Comment:
nit: we dont display "using an astro api token" only on deployment inspect right? maybe we should rename the variable to check `isDeploymentInspect`